### DISCUSSION TITLE
Megasync app loaded in systray, but can't open login dialog

### DISCUSSION THREAD

**Discussion by tone-milazzo** [i3/i3#6125] [Q&A]
I got megasync to start on boot with
`exec --no-startup-id megasync`
but there's no CLI login. The only way to login is to left click on the applet, then click on "Log IN" to open the login dialog. And that dialog might be manifesting as this tiny little sliver of a window. Like, 3px wide  by 15px high. I assume that's it
Similarly, the Setting dialog opens from the applet's left click menu. But then immediately opens the Proxy sub dialog, blocking the Setting dialog.

**Comment by tone-milazzo** [comment]
Just realized I was able to grab the corner of that tiny dialog and drag it open. Which is good, but I'm wondering if this is a sign of something I'll need to fix now for future applets.

**Reply by orestisfl** [reply]
A workaround would be to have a `for_window` rule to resize & reposition the window every time it opens, see the userguide. Can't investigate why that happens right now.
